mavisp/modules.py

```diff
@@ -237,7 +237,29 @@ def ingest(self, mutations):
         else:
             rasp_header = None
 
-        # check if we have both FoldX and Rosetta/RaSP col
+        # Add mean of FoldX and Rosetta (Foldetta)
+        if foldx_header in model_data.columns and rosetta_header in model_data.columns:
+            both_valid = model_data[[foldx_header, rosetta_header]].notna().all(axis=1)
+            foldetta_mean = 'Foldetta Mean (FoldX, Rosetta)'
```

Comment:
can we call this column similarly to other Stability columns? e.g. for FoldX we have

```
Stability (FoldX5, alphafold, kcal/mol)
```

the equivalent for Foldetta would be

```
Stability (Foldetta from FoldX and Rosetta, alphafold, kcal/mol)
```

same idea for the classification columns